ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-navigation-test.js

```diff
@@ -1438,6 +1438,16 @@ path "${this.backend}/*" {
       assert.dom(PAGE.secretTab('Version History')).doesNotExist('Version History tab not shown');
 
       await click(PAGE.secretTab('Secret'));
+      assert.true(
+        await waitUntil(() => currentRouteName() === 'vault.cluster.access.control-group-accessor'),
+        'redirects to access control group route'
+      );
+      await grantAccess({
+        apiPath: `${backend}/data/${encodeURIComponent(secretPath)}`,
+        originUrl: `/vault/secrets/${backend}/kv/${secretPathUrlEncoded}/details`,
+        userToken: this.userToken,
+        backend: this.backend,
+      });
```

Comment:
want to call out this change. I'm pretty sure that because the request to `/data/` has moved down we are prompted with another control group error here (after clicking the secret tab) and need to grant access. Want a gut check here that this makes sense!

Reply:
originURL is going to be the page they were on before they got to the page that requested the control group resource (so in this case, it's `/vault/secrets/${backend}/kv/${secretPathUrlEncoded}/paths`). Then the user needs to click the secret tab again and ensure that the secret data loads correctly.

Reply:
hmm, I had that originally and the test failed. 🤔  I'll try again

Reply:
I think in my original attempt I had forgotten the `waitUntil`!